packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -251,13 +251,25 @@ Future<XcodeBuildResult> buildXcodeProject({
   );
 
   if (hasPlugins()) {
-    final String currentGeneratedXcconfig = readGeneratedXcconfig(app.appDirectory);
+    final String iosPath = fs.path.join(fs.currentDirectory.path, app.appDirectory);
+    // If the Xcode project, Podfile, or Generated.xcconfig have changed since
+    // last run, pods should be updated.
+    final Fingerprinter fingerprinter = new Fingerprinter(
+      fingerprintPath: fs.path.join(getIosBuildDirectory(), 'pod_inputs.fingerprint'),
+      paths: <String>[
+        _getPbxProjPath(app.appDirectory),
+        fs.path.join(iosPath, 'Podfile'),
+        fs.path.join(iosPath, 'Flutter', 'Generated.xcconfig'),
+      ],
+      properties: <String, String>{},
```

Comment:
can we add some dartdocs to fingerprint.dart or tweak the name? I couldn't figure out what this does. Feel free to do in a different PR

Reply:
You answered this one

Reply:
Will do. The intent for properties is to be build options/flags (e.g. build mode, target platform, etc.) it's marked as `@required` in Fingerprinter because in almost every case this is something we don't want to forget, but in this case there's not much to put in there.